### PR TITLE
I have moved the images in the article to the correct positions.

### PR DESCRIPTION
**Pull request by DWJ-Squirtle**
I have discovered that the storage location of the images on my blog was incorrect, and I have now corrected it.

### CODE REVIEW

File: blog/Moodle in Kubeblocks（windows）.md [hunk 1]
```diff
@@ -105,25 +105,25 @@ Open a browser and enter the address localhost/moodle, then follow the steps bel
...
 
-![](https://raw.githubusercontent.com/DWJ-Squirtle/blogimage/main/choose_language.png)
+![](../static/images/moodle-choose_language.png)
 
 #### 2.confirm paths
 
 Default configuration is fine, just click next
 
-![](https://raw.githubusercontent.com/DWJ-Squirtle/blogimage/main/confirm%20paths.png)
+![](../static/images/moodle-confirm%20paths.png)
```
Comment (michelle-0808) [inline blog/Moodle in Kubeblocks（windows）.md:114]: Use a hyphen instead of a blank space between words regarding the name of an image.

Comment (DWJ-Squirtle) [reply blog/Moodle in Kubeblocks（windows）.md]: ok

**Review by michelle-0808 — APPROVED** [review]

**Comment by michelle-0808** [comment]
Hi @DWJ-Squirtle, 
There are some conflicts you need to resolve first. 
Remember to run `git pull` before any development.

**Comment by DWJ-Squirtle** [comment]
> Hi @DWJ-Squirtle, There are some conflicts you need to resolve first. Remember to run `git pull` before any development.

Thank you for the reminder, I will pay attention to it.

**Pull request merged**